### DISCUSSION TITLE
I hope to display vertical lines

### DISCUSSION THREAD

**Discussion by Sakurazuka1027** [mfcc64/youtube-musical-spectrum#8] [Ideas]
Hello.
Thank you for the good feature.
I use it to transcription for electric bass.
However, Bass sound's width is wide, it is difficult for me to discerning scales.
Therefor, I hope to display vertical lines.
![sp_with_vlines](https://github.com/mfcc64/youtube-musical-spectrum/assets/161039756/c0294658-0c98-4177-bf27-4624de6fe122)

**Comment by mfcc64** [comment]
Try this code on browser console:
```js
document.getElementsByTagName("showcqt-element")[0]
.actual_render_callback = function(color) {
    const len = color.length / 4;
    const types = [0,1,1,0,1,0,1,0,1,1,0,1];
    let last = 0;
    for (let x = 0; x < len; x++) {
        const current = Math.round(x * 120 / len);
        if (current != last) {
            last = current;
            const type = types[current % 12];
            color[4*x+0] = type;
            color[4*x+1] = !type;
            color[4*x+2] = type;
            color[4*x+3] = 100;
        }
    }
};
```

I'm unsure if I should add this feature or not.

Thank's
